tests/phpunit/tests/themes/test-twenty-fourteen.php

```diff
@@ -2,23 +2,18 @@
 
 class Twenty_Fourteen_Test extends PLL_UnitTestCase {
 	protected static $stylesheet;
-	protected static $tag_en;
-	protected static $tag_fr;
+	protected static $tags;
```

Comment:
No sure this property is necessary. A local variable should be fine.
Indeed, I don't see it populated in a static method such as `pllSetUpBeforeClass()` (or maybe I'm missing something).